tests/unit/jbi/test_bugzilla.py

```diff
@@ -0,0 +1,48 @@
+"""
+Module for testing src/jbi/configuration.py functionality
+"""
+# pylint: disable=cannot-enumerate-pytest-fixtures
+import pytest
+
+from src.jbi import bugzilla
+
+
+def test_get_jira_labels_without_whiteboard():
+    test_bug = bugzilla.BugzillaBug(id=0, whiteboard="")
+    jira_labels = test_bug.get_jira_labels()
+    assert jira_labels
```

Comment:
I would say that this assertion is useless. py.test provides a pretty good output if you compare a falsy value with a list

(same with other tests)

Reply:
https://github.com/mozilla/jira-bugzilla-integration/pull/59#discussion_r871825582

resolving